features/polyExplorer/src/components/clusterStory/clusterStory.css

```diff
@@ -64,7 +64,7 @@
 }
 
 .cluster-story .cluster-story-title {
-    color: var(--story-color-primary) !important;
```

Comment:
Nice! Should be getting rid of all of these if we see some :)

Reply:
I guess we should take a look at each case in particular. Here, for example, it was possible to delete the `!important` because we implemented the theming and the "cascading hierarchy" for the font colors changed.

Reply:
Yeah totally! I think if we remove all * selectors we could go with natural cascading